data/style.css

```diff
@@ -6,6 +6,7 @@ body {
 	padding: 0 16px ;
 	margin-bottom: 500px ;
 	font-family: sans-serif ;
+	scrollbar-color: white #151515;
 }
 
```

Comment:
```suggestion
::-webkit-scrollbar {
    height: 12px;
    width: 12px;
    background: #151515;
}

::-webkit-scrollbar-thumb {
    background: #000;
    -webkit-box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.75);
}

::-webkit-scrollbar-corner {
    background: #000;
}

```

Reply:
This is actually a bad idea as it is, as zooming into page also bloody makes the scroll bar zoom in, horrid. Bear with me

Reply:
Yeah, I would disregard using webkit retrospectively. If only other browsers were as graceful as Firefox.
How does Github make the dark scrollbar?

Reply:
Haven't tried anything, but you can see what github does in the browser devtools:

```css
::-webkit-scrollbar {
    background-color: #202324;
    color: #aba499;
}

::-webkit-scrollbar-corner {
    background-color: #181a1b;
}

::-webkit-scrollbar-thumb {
    background-color: #454a4d;
}
```

Reply:
> Haven't tried anything, but you can see what github does in the browser devtools

Where did you find that? I was looking, but haven't found anything. What element did you select and where did you look?

Reply:
> Haven't tried anything, but you can see what github does in the browser devtools:
> 
> ```css
> ::-webkit-scrollbar {
>     background-color: #202324;
>     color: #aba499;
> }
> 
> ::-webkit-scrollbar-corner {
>     background-color: #181a1b;
> }
> 
> ::-webkit-scrollbar-thumb {
>     background-color: #454a4d;
> }
> ```

I have updated my suggested edit to use this, the Github colour codes, thanks.

Reply:
Should I use the same color codes as GitHub light mode too? And how did you guys find these selectors? They don't show up anywhere for me??

Reply:
Well since webkit will work for firefox too I assume, the scrollbar-width property is redundant. And IDK how Hallzy found the codes for githubs colour.

Reply:
> Well since webkit will work for firefox too I assume, I would say the scrollbar-width property is redundant.

Uhm no, all these `::-webkit-` prefixed pseudoselectors don't affect FF at all. I don't see stylized scrollbars in GH with FF. Anyway, I also don't see stylized scrollbars for GH in Chrome either, I just realized, I just see the dark scrollbar, bc GH has `color-scheme: dark`.

Can anyone of you pls send a screenshot of the scrollbar or the `::-webkit-` selector on GH? I can't find it, nor the colors from there. I would need those for the light mode styling.